content/v0.79.0.md

```diff
@@ -436,7 +436,7 @@ dos_rate = default | host | custom-rate
 
 ###  5.1. <a name='PipingfromtheDOSPromptorBatchFiles'></a>Piping from the DOS Prompt or Batch Files
 
-DOSBox did not support the piping operation previously, but DOSBox Staging now brings piping support as how it has worked in DOS and most other operating systems. This allows two or more commands to communicate by passing the output text of one command to another as its input text. For example, commands like `TYPE FILE.TXT | MORE` will work, so that you will be able to display the content of `FILE.TXT` one screen at a time, Multiple piping is also supported, such as `DIR | SORT | MORE` for displaying sorted directory output one screen at a time.
+DOSBox did not support the piping operation previously, but DOSBox Staging now brings piping support as how it has worked in DOS and most other operating systems. This allows two or more commands to communicate by passing the output text of one command to another as its input text. For example, commands like `ECHO Y | CHOICE` will work, so that Y option will be passed as the input of CHOICE command. Multiple piping is also supported, such as `DIR | SORT | MORE` for displaying sorted directory output one screen at a time (provided that you have `SORT` and `MORE` commands from MS-DOS or FreeDOS in this case). If the current directory and `C:\` are both read-only, the environment variable `%TEMP%` (or `%TMP%`) need to be set within DOS pointing to a writable directory so that piping will work properly (e.g. `SET TEMP=C:\TEMP`).
```

Comment:
the `Y` option
input of the `CHOICE` command
Multiple piping -> chained piping
the `SORT` and `MORE` commands
drop "in this case", replace with "in your path"
needs to be set within DOS to point to a writable

Reply:
Thanks for the suggestions. Done modifying all of them.